src/main/java/io/gravitee/policy/v3/apikey/ApiKeyPolicyV3.java

```diff
@@ -78,6 +78,10 @@ public void onRequest(Request request, Response response, ExecutionContext execu
             final String apiId = (String) executionContext.getAttribute(ExecutionContext.ATTR_API);
 
             Optional<ApiKey> apiKeyOpt = executionContext.getComponent(ApiKeyService.class).getByApiAndKey(apiId, requestApiKey);
+
+            // Always set the apikey into the context
+            executionContext.setAttribute(ATTR_API_KEY, requestApiKey);
```

Comment:
Should we also align the behavior for the V4 version (eg ApiKeyPolicy)?
What is the official meaning of the apikey attribute currently? What if people currently expect to have it only in case the apikey is valid (which seems to be the current behavior)? Should it be documented as a breaking change?